### ISSUE TITLE
Remove soma_pcl_segmentation for release

### ISSUE THREAD

**Issue by kunzel** [strands-project/soma#71]
However, keep it on a branch if needed later...

**Comment by hkaraoguz** [comment]
@kunzel I guess this issue is resolved

**Comment by nilsbore** [comment]
Before we all forget about this, a lot of tools are using SOMA as a dependency and the `semantic_segmentation` dependency of `soma_pcl_segmentation` is not easily installable. I would like to refer to the github instead of the prebuilt archives since it's easier than setting up another package repository. Can we remove the `soma_pcl_segmentation` from the master branch?

**Comment by hawesie** [comment]
+1

**Comment by hkaraoguz** [comment]
I will prepare a new version of soma next week and remove that package. @jayyoung Can you send a pr if you have any updates for the next release?

**Comment by jayyoung** [comment]
Yeah I have a couple of really super minor tweaks to make, and I'll add them with a PR on Monday sometime.
